src/main/java/com/powsybl/openloadflow/ac/equations/AcEquationSystemCreator.java

```diff
@@ -216,7 +216,8 @@ static <T extends LfElement> void updateRemoteVoltageControlEquations(VoltageCon
                 .map(vc -> equationSystem.getEquation(vc.getControlledBus().getNum(), AcEquationType.BUS_TARGET_V).orElseThrow())
                 .collect(Collectors.toList());
 
-        if (controlledBus.isDisabled() || voltageControl.isDisabledAndAlsoAllItsDependentVoltageControls()) {
```

Comment:
I disagree with this change. All of this is just a question of having or not having a PV equation at controlled bus (or in the zero impedance network of the controlled bus).
- See the unit test I have added, we deactivate a PV equation. All the controllers are disabled, but we must keep the PV equation because we have a generator.
- Your change is really different than before because you only consider disabled voltage control with the same priority. But with `isDisabledAndAlsoAllItsDependentVoltageControls`, we are looking of all the voltage controls of the zero impedance network, with different priority.

Reply:
````
if (voltageControl.isHidden()) {
            if (condition) { // condition that we have to find together
                      vEq.setActive(false);
            }
            for (T controllerElement : controllerElements) {
                equationSystem.getEquation(controllerElement.getNum(), distrEqType)
                        .orElseThrow()
                        .setActive(false);
                equationSystem.getEquation(controllerElement.getNum(), ctrlEqType)
                        .orElseThrow()
                        .setActive(true); 
} else {
          List<T> enabledControllerElements = controllerElements.stream()
                        .filter(voltageControl::isControllerEnabled).collect(Collectors.toList());
          List<T> disabledControllerElements = controllerElements.stream()
                        .filter(Predicate.not(voltageControl::isControllerEnabled)).collect(Collectors.toList());

           // activate voltage control at controlled bus only if at least one controller element is enabled
           vEq.setActive(!enabledControllerElements.isEmpty());
}
````
Something like that is much more coherent with all the rework we are doing in outer loops #840 
What do you think ?